app/controllers/collections_controller.rb

```diff
@@ -39,6 +42,61 @@ def show
     end
   end
 
+  def create
+    if request.format == 'json' and request.post?
+      collection_name = params[:name]
+      if (!collection_name.nil? and !collection_name.blank? and !(collection_name.length > 255) and !(params[:collection_metadata].nil?))
+        # Parse JSON-LD formatted collection metadata and convert to RDF
+        json_md = params[:collection_metadata]
+        graph = RDF::Graph.new << JSON::LD::API.toRdf(json_md)
+        collection_rdf = graph.dump(:ttl, prefixes: {foaf: "http://xmlns.com/foaf/0.1/"})
+        collection_uri = graph.statements.first.subject.to_s
+        if !Collection.find_by_uri(collection_uri).present?  # ingest skips collections with non-unique name or uri
+          # write metadata and manifest file for ingest
+          collection_dir = NEW_COLLECTION_DIR
```

Comment:
why are you creating another variable here, can't you just use `NEW_COLLECTION_DIR` directly?